docker/initialize-reaper_db/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM cassandra:3.11
+
+# create the required reaper_db keyspace to allow Reaper to store scheduling data
+CMD set -x \
+    && cqlsh cassandra -e \
+    "CREATE KEYSPACE IF NOT EXISTS reaper_db WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1 };"
```

Comment:
this RF (i presume) it only for the one-node-cassandra within the docker container.

Maybe it's the applicaton's responsibility to log a warning at startup if RF=1 and N>1
wdyt?